City-of-Helsinki/drupal-hdbt-admin
# UHF-2076 CKEditor link

How to test (if you have no setup):
- `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
- `cd hel-platform`
- Switch to corresponding branches: 
  - `composer require drupal/hdbt_admin:dev-UHF-2076_ckeditor_link`
- Run `make new`
- Meanwhile, check this video how the link plugin works now: https://user-images.githubusercontent.com/1712902/134370454-ef8be42b-255d-43ce-a808-2f2d153890b8.mov

- Log in and go to add/edit any page.
   - Add a text paragraph and create a link with the ckeditor link plugin. Select protocol for the link, fill in the fields and save the page.
   - Try to reproduce the bug from the video.
